var/spack/repos/builtin/packages/py-pymol/package.py

```diff
@@ -14,18 +14,29 @@ class PyPymol(PythonPackage):
 
     homepage = "https://pymol.org"
     url      = "https://github.com/schrodinger/pymol-open-source/archive/v2.3.0.tar.gz"
+    url      = "https://github.com/schrodinger/pymol-open-source/archive/v2.4.0.tar.gz"
```

Comment:
Only one URL is needed here, Spack contains magic to figure out what the URL is for each version. Also, we usually sort versions from newest to oldest.